test/validation/api/ipsec/ipsec.c

```diff
@@ -1155,6 +1163,52 @@ int ipsec_config(odp_instance_t ODP_UNUSED inst)
 	ipsec_config.inbound.chksums.all_chksum = ~0;
 	ipsec_config.stats_en = true;
 
+	ipsec_config.inbound.reassembly.max_wait_time = 20 * ODP_TIME_MSEC_IN_NS;
```

Comment:
The value configured should not exceed the max_wait_time capability which is not checked here. The code should cap the configured value to the max_wait_time capability.

Another thing I wonder is whether 20 ms is a bit tight considering that the validation tests get run in various virtual environments that may share CPU cores with other things and suffer from scheduling delays etc. Something like 100 ms (or the max supported if max_wait_time capability is less) would seem better to me.

Reply:
I can change it to MIN(100ms, max_wait_time_capability). Setting directly to max_wait_time would mean system would wait for huge duration (if there is capability for large wait times).